manifests/dev/deployment.yml

```diff
@@ -28,23 +28,28 @@ spec:
             path: /
             port: 80
           initialDelaySeconds: 3
-          periodSeconds: 3
+          periodSeconds: 60
+          successThreshold: 1
+          failureThreshold: 3
+          timeoutSeconds: 60
         resources:
           requests:
-            cpu: 250m
+            cpu: 100m
+            memory: 2000M
           limits:
-            cpu: 750m   
-            memory: 2048M
+            cpu: 250m   
+            memory: 4000M
```

Comment:
and I'd leave low limit too, if we do hit a limit, I'd rather catch it in dev more carefully